diff --git a/node_modules/workerd/worker.mjs b/node_modules/workerd/worker.mjs
index c6ef576..8b6d910 100755
--- a/node_modules/workerd/worker.mjs
+++ b/node_modules/workerd/worker.mjs
@@ -8877,7 +8877,7 @@ interface BasicImageTransformations {
    *   in combination with the properties above
    * - border - this will automatically trim the surroundings of an image based on
    *   it's color. It consists of three properties:
-   *    - color: rgb or hex representation of the color you wish to trim (todo: verify the rgba bit)
+   *    - color: rgb or hex representation of the color you wish to trim (rgba is also supported)
    *    - tolerance: difference from color to treat as color
    *    - keep: the number of pixels of border to keep
    */
